src/gazebo_geotagged_images_plugin.cpp

```diff
@@ -555,11 +566,25 @@ void GeotaggedImagesPlugin::_handle_request_camera_settings(const mavlink_messag
         &msg,
         0,                      // time_boot_ms
         CAMERA_MODE_IMAGE,      // Camera Mode
-        NAN,                    // Zoom level
+        100.0 *(_zoom - 1.0)/ (_maxZoom - 1.0),                    // Zoom level //TODO: Output zoom
```

Comment:
```suggestion
        1.0E2 * (_zoom - 1.0) / (_maxZoom - 1.0),            // Zoom level //TODO: Output zoom
```
Also can you explain whar exactly is the TODO here?

Reply:
Ah, sorry that should not have been there